.env.development

```diff
@@ -0,0 +1,8 @@
+#ULOAD-REQUEST
+VITE_UPDATE_UPLOAD_REQUEST_TIME=6000
+
+#LINK-GEY-UPLOAD-REQUEST
```

Comment:
Do we really need this comment? I understand the var name even without a comment :)